src/ballet/lthash/fd_lthash.h

```diff
@@ -54,6 +57,69 @@ fd_lthash_sub( fd_lthash_value_t * restrict       r,
   return r;
 }
 
+static inline void
+fd_lthash_hash( fd_lthash_value_t const *  r, uchar hash[ static 32] ) {
+  ulong *p = (ulong *) r->bytes;
+  for ( ulong i=0; i<(FD_LTHASH_LEN_BYTES / sizeof(ulong)); i++ ) {
+    if (*p++ != 0) {
+      fd_blake3_t b3[1];
+      fd_blake3_init  ( b3 );
+      fd_blake3_append( b3, r->bytes, FD_LTHASH_LEN_BYTES  );
+      fd_blake3_fini  ( b3, hash );
+      return;
+    }
+  }
+
+  fd_memset(hash, 0, 32);
+}
+
+#define FD_LTHASH_ENC_32_BUF( _x, _y ) __extension__({                   \
+  if( FD_UNLIKELY( _x == NULL ) ) {                                      \
+    strcpy(_y, "<NULL>");                                                \
+  } else {                                                               \
+    uchar _hash[32];                                                     \
+    fd_lthash_hash(_x, _hash);                                           \
+    fd_base58_encode_32( _hash, NULL, _y );                              \
+  }                                                                      \
+})
+
+#define FD_LTHASH_ENC_32_ALLOCA( x ) __extension__({                     \
+  char *_out = (char *)fd_alloca_check( 1UL, FD_BASE58_ENCODED_32_SZ );  \
+  FD_LTHASH_ENC_32_BUF(x, _out);                                         \
+  _out;                                                                  \
+})
+
```

Comment:
Should this call `fd_lthash_fini` and log that? Might make the hashes easier to compare

Reply:
This was the shortened hash anza wanted.  Yes, it is not 2k worth of data but this is just to help visually compare we got the same hash